docs/usage.md

```diff
@@ -5,7 +5,7 @@ The SEO tag will respect any of the following if included in your site's `_confi
 * `title` - Your site's title (e.g., Ben's awesome site, The GitHub Blog, etc.)
 * `description` - A short description (e.g., A blog dedicated to reviewing cat gifs)
 * `url` - The full URL to your site. Note: `site.github.url` will be used by default.
-* `author` - global author information (see below)
+* `author` - global author information (see [Advanced usage](https://github.com/jekyll/jekyll-seo-tag/blob/master/docs/advanced-usage.md#author-information))
```

Comment:
If we write this as `advanced-usage.md#author-information`, it should work on both GitHub.com and GitHub Pages.